mantidproject/mantid
# 27638 sample logs

This fixes two minor bugs. The plotting one was discovered while working on the missing attribute in the processed nexus files.

**To test:**

Run the script in #27638 and see that it now passes. Also "plot" (they are single value so there isn't much to see) both logs in mantidworkbench.

Fixes #27638

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
